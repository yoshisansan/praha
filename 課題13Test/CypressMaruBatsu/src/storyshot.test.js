import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({ suite: 'Storyshots' });
