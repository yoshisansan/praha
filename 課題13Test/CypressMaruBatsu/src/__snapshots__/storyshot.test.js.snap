// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Board Batsu 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square0"
        onClick={[Function]}
      >
        ×
      </button>
      <button
        className="square"
        id="square1"
        onClick={[Function]}
      >
        ×
      </button>
      <button
        className="square"
        id="square2"
        onClick={[Function]}
      >
        ×
      </button>
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square3"
        onClick={[Function]}
      >
        ×
      </button>
      <button
        className="square"
        id="square4"
        onClick={[Function]}
      >
        ×
      </button>
      <button
        className="square"
        id="square5"
        onClick={[Function]}
      >
        ×
      </button>
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square6"
        onClick={[Function]}
      >
        ×
      </button>
      <button
        className="square"
        id="square7"
        onClick={[Function]}
      >
        ×
      </button>
      <button
        className="square"
        id="square8"
        onClick={[Function]}
      >
        ×
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Board Default 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square0"
        onClick={[Function]}
      />
      <button
        className="square"
        id="square1"
        onClick={[Function]}
      />
      <button
        className="square"
        id="square2"
        onClick={[Function]}
      />
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square3"
        onClick={[Function]}
      />
      <button
        className="square"
        id="square4"
        onClick={[Function]}
      />
      <button
        className="square"
        id="square5"
        onClick={[Function]}
      />
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square6"
        onClick={[Function]}
      />
      <button
        className="square"
        id="square7"
        onClick={[Function]}
      />
      <button
        className="square"
        id="square8"
        onClick={[Function]}
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Board Maru 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square0"
        onClick={[Function]}
      >
        ○
      </button>
      <button
        className="square"
        id="square1"
        onClick={[Function]}
      >
        ○
      </button>
      <button
        className="square"
        id="square2"
        onClick={[Function]}
      >
        ○
      </button>
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square3"
        onClick={[Function]}
      >
        ○
      </button>
      <button
        className="square"
        id="square4"
        onClick={[Function]}
      >
        ○
      </button>
      <button
        className="square"
        id="square5"
        onClick={[Function]}
      >
        ○
      </button>
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square6"
        onClick={[Function]}
      >
        ○
      </button>
      <button
        className="square"
        id="square7"
        onClick={[Function]}
      >
        ○
      </button>
      <button
        className="square"
        id="square8"
        onClick={[Function]}
      >
        ○
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Board Sankaku 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square0"
        onClick={[Function]}
      >
        △
      </button>
      <button
        className="square"
        id="square1"
        onClick={[Function]}
      >
        △
      </button>
      <button
        className="square"
        id="square2"
        onClick={[Function]}
      >
        △
      </button>
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square3"
        onClick={[Function]}
      >
        △
      </button>
      <button
        className="square"
        id="square4"
        onClick={[Function]}
      >
        △
      </button>
      <button
        className="square"
        id="square5"
        onClick={[Function]}
      >
        △
      </button>
    </div>
    <div
      className="board-row"
    >
      <button
        className="square"
        id="square6"
        onClick={[Function]}
      >
        △
      </button>
      <button
        className="square"
        id="square7"
        onClick={[Function]}
      >
        △
      </button>
      <button
        className="square"
        id="square8"
        onClick={[Function]}
      >
        △
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Game Default 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <div>
      <div
        className="board-row"
      >
        <button
          className="square"
          id="square0"
          onClick={[Function]}
        />
        <button
          className="square"
          id="square1"
          onClick={[Function]}
        />
        <button
          className="square"
          id="square2"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          id="square3"
          onClick={[Function]}
        />
        <button
          className="square"
          id="square4"
          onClick={[Function]}
        />
        <button
          className="square"
          id="square5"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          id="square6"
          onClick={[Function]}
        />
        <button
          className="square"
          id="square7"
          onClick={[Function]}
        />
        <button
          className="square"
          id="square8"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
  <div
    className="game-info"
  >
    <div>
      次のプレイヤーは: X
    </div>
    <ol>
      <li>
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;

exports[`Storyshots Square Batsu 1`] = `
<button
  className="square"
  id="squareundefined"
>
  ×
</button>
`;

exports[`Storyshots Square Default 1`] = `
<button
  className="square"
  id="squareundefined"
/>
`;

exports[`Storyshots Square Maru 1`] = `
<button
  className="square"
  id="squareundefined"
>
  ○
</button>
`;

exports[`Storyshots Square Sankaku 1`] = `
<button
  className="square"
  id="squareundefined"
>
  △
</button>
`;
